### ISSUE TITLE
Unintended highlight with e_map timeline

### ISSUE THREAD

**Issue by psimm** [JohnCoene/echarts4r#272]
When a chart has a choropleth map and a timeline, clicking on the timeline dots highlights countries. 
This is a new behavior that does not occur on the last CRAN release.

Example:

```
library(dplyr)
library(echarts4r)

choropleth <- data.frame(
  countries = rep(c("United States", "Russia", "Brazil"), 2),
  date = rep(c(as.Date("2021-01-01"), as.Date("2021-01-02")), each = 3),
  values = runif(6)
)

choropleth %>%
  group_by(date) %>%
  e_charts(countries, timeline = TRUE) %>%
  e_map(values) %>%
  e_visual_map(min = 0, max = 1)
```

On initial load:
<img width="523" alt="Screen Shot 2021-02-10 at 11 53 40" src="https://user-images.githubusercontent.com/25177095/107500584-9f7d7d80-6b96-11eb-8147-2110ef7b3c8c.png">

After clicking the second timeline dot (and not clicking on Russia):
<img width="532" alt="Screen Shot 2021-02-10 at 11 50 08" src="https://user-images.githubusercontent.com/25177095/107500148-2120db80-6b96-11eb-9ceb-56cb94c3532a.png">

Clicking on the arrows does not trigger the highlight. It seems that the timeline dot has an unintentional connection to the other plot data.

This behavior can also be seen on the website: https://echarts4r.john-coene.com/articles/timeline.html#general-options

**Comment by JohnCoene** [comment]
Interesting. That seems to be from echarts.js itself as far as I can tell but I will have to dig in a little more.

**Comment by jordancampbell23** [comment]
I was able to get rid of this by setting

`e_timeline_opts(
        axis_type = "time"
)`

Hope that works!

**Comment by JohnCoene** [comment]
Funny, I received an email notification from Github about a comment from @jordancampbell23 on `e_timeline_opts( axis_type = "time")` fixing the issue but I don't see the comment on here.

**Comment by jordancampbell23** [comment]
Yes, my bad. I got rid of a similar issue on another timeline--but it didn't resolve this issue.

**Comment by JohnCoene** [comment]
No problem @jordancampbell23  :)

**Comment by helgasoft** [comment]
[recognized as bug](https://github.com/apache/echarts/issues/14931) in ECharts v.5+

**Comment by jordancampbell23** [comment]
@JohnCoene one way around the distraction of this bug is to disable select in e_map by adding the following:

`select = list(disabled = T)`

I don't find the select feature adds much to the maps, so personally it has minimal downside.

**Comment by JohnCoene** [comment]
Thank you for sharing!
